src/Microsoft.Health.Fhir.SqlServer/Features/Search/SearchParameterColumnLocation.cs

```diff
@@ -0,0 +1,19 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.Search
+{
+    /// <summary>
+    /// Indicates where the column(s) of for a search parameter are located.
```

Comment:
>of for [](start = 38, length = 6)

super nit: two prepositions together look weird for me, but English is not my native.

Reply:
Ha ha, its definitely not proper English! Will fix.